runtime/jcl/common/java_lang_Class.cpp

```diff
@@ -854,6 +854,15 @@ Java_java_lang_Class_getEnclosingObject(JNIEnv *env, jobject recv)
 							resultObject = vm->reflectFunctions.createDeclaredConstructorObject(method, resolvedClass, NULL, currentThread);
 						}
 					}
+#if JAVA_SPEC_VERSION >= 25
+					else {
+						/* There is an enclosing constructor or method from getEnclosingMethodForROMClass(),
+						 * but it can't be found, return its descriptor string for error handling.
+						 */
+						resultObject = vm->memoryManagerFunctions->j9gc_createJavaLangString(currentThread,
+								J9UTF8_DATA(enclosingMethodSigUTF), J9UTF8_LENGTH(enclosingMethodSigUTF), 0);
```

Comment:
> https://github.com/eclipse-omr/omr/blob/3223d8d0d584575eafb5ec462250baae92cd6084/doc/CodingStandard.md#function-parameter-list-does-not-fit-on-one-line

Start the parameter list on a new line. 
```suggestion
						resultObject = vm->memoryManagerFunctions->j9gc_createJavaLangString(
										currentThread, J9UTF8_DATA(enclosingMethodSigUTF),
										J9UTF8_LENGTH(enclosingMethodSigUTF), 0);
```

Reply:
Moved the first parameter.